dev-tools/ecs-migration.yml

```diff
@@ -119,402 +100,613 @@
   to: process.executable
```

Comment:
Should we have an explicit entry like `beat: *`? I'm ok if not.

Reply:
For now my plan was to check if it is there at all or empty, instead of having a special char. Will update in case we need it differently.